src/analysis/retail/priest/shadow/modules/guide/CooldownGraphSubsection.tsx

```diff
@@ -75,7 +82,7 @@ const CoreCooldownsGraph = () => {
       VoidboltAnalyzer?.executeRanges.map(fromExecuteRange);
 
     message = (
-      <Trans id="guide.priest.shadow.sections.corecooldowns.graphVB2">
+      <Trans id="guide.priest.shadow.sections.corecooldowns.graphVB">
```

Comment:
We're abandoning translations inside specs, feel free to remove this wrapper

Reply:
I'll start going through and removing them from shadow's guide and statistics.